### PR TITLE
Prepare to release the next wave of providers:

### PR DESCRIPTION
**Pull request by potiuk**
Regular providers:

* amazon          - 1.2.0
* apache.beam     - 1.0.1
* apache.druid    - 1.1.0
* apache.hive     - 1.0.2
* apache.spark    - 1.0.2
* cncf.kubernetes - 1.0.2
* dingding        - 1.0.2
* docker          - 1.0.2
* elasticsearch   - 1.0.2
* exasol          - 1.1.0
* google          - 2.1.0
* http            - 1.1.1
* jenkins         - 1.1.0
* microsoft.azure - 1.2.0
* mysql           - 1.0.2
* neo4j           - 1.0.1
* openfaas        - 1.1.1
* papermill       - 1.0.2
* presto          - 1.0.2
* qubole          - 1.0.2
* salesforce      - 2.0.0
* sendgrid        - 1.0.2
* sftp            - 1.1.1
* slack           - 3.0.0
* snowflake       - 1.1.1
* sqlite          - 1.0.2
* ssh             - 1.2.0
* tableau         - 1.0.0
* telegram        - 1.0.2

Backport providers (all in version 2021.3.3):

amazon apache.beam apache.druid apache.hive apache.spark cncf.kubernetes
dingding docker elasticsearch exasol google http jenkins microsoft.azure
mysql neo4j openfaas papermill presto qubole salesforce sendgrid sftp
slack snowflake sqlite ssh tableau telegram

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by RosterIn** [comment]
What about Amazon & Azure?
looking at the modified files It doesn't appear in the list however it seems it is going to be released but not for backported?

**Comment by potiuk** [comment]
> What about Amazon & Azure?
> looking at the modified files It doesn't appear in the list however it seems it is going to be released but not for backported?

oh yeah

**Comment by potiuk** [comment]
> > What about Amazon & Azure?
> > looking at the modified files It doesn't appear in the list however it seems it is going to be released but not for backported?
> 
> oh yeah

Fixed. We release backports for those as well :)

**Comment by RosterIn** [comment]
is it too late for https://github.com/apache/airflow/pull/11015 to be included?

**Comment by potiuk** [comment]
> is it too late for #11015 to be included?

Not too late. I just rebased and regenerated all documents.

**Comment by potiuk** [comment]
Refreshed, rebased to latest changes and included all the providers with doc-only changes (this way documentation for them will get also updated ).  Once this one passes and we merge, I am happy to cut RCs for all the providers @ashb @kaxil

**Comment by potiuk** [comment]
@ashb  if you do not feel ok with it - I am not asking you to approve that one, but I hope this one will not get blocked for too long - there are at least few people who are waiting for the new providers to be released.

**Comment by ashb** [comment]
I have no problem with someone else doing it, just not comfortable putting my name as relases manager for something I haven't tested. Reviewing

File: airflow/providers/apache/hive/CHANGELOG.rst
Comment (ashb) [inline airflow/providers/apache/hive/CHANGELOG.rst:28]: Is it actually worth releasing this if it's the only change?

File: airflow/providers/docker/CHANGELOG.rst
Comment (ashb) [inline airflow/providers/docker/CHANGELOG.rst:28]: Ditto

**Review by ashb — COMMENTED** [review]
We have a number of providers where the only change is #14082 - that feels like it's not worth releasing as out doesn't change anything from a user doc or code PoV right?

(No harm to users in it. :shrug:)

We've never done it before, but Python does have a convention for "this is a doc only changes" `.post1`. I imagine most of our tooling and process works break in such a version though

File: airflow/providers/apache/hive/CHANGELOG.rst
Comment (potiuk) [reply airflow/providers/apache/hive/CHANGELOG.rst:28]: I believe so. Since it costs us nothing but it improves the documentation which is released as part of the providers - yes I thought about it yesterday and while I initially excluded them, I think it is definitely worth to release it. It is important to get the updated user documentation in the hands of the users I believe. 

It has zero risk, practically no consequences for our users (they will just install newer version when they do). It costs us almost nothing for the batch release. 

Yes. I truly believe it is worth to release them.

**Comment by potiuk** [comment]
> We've never done it before, but Python does have a convention for "this is a doc only changes" `.post1`. I imagine most of our tooling and process works break in such a version though

we could potentially do it this way, but I think there is indeed no harm and almost no cost.

Also there is a benefit for release manager. Next time packages are prepared you will not see those as "potential release candidates" that saves time next time when we prepare release. If only that is a benefit - I would do it anyway.

Alternative to that is mark certain commits (like this one) as "not-worthy-to-be-released" and exclude those when preparing pacakges. Which we might want to do in the future. I think that should be part of bigger change which we might consider. Right now it is release manager who splits the changes to bugs/fixes/doc changes etc.  But there was a discussion about using semantic commits or labels for those. We might want to resurrect that discussion (though I think it is perfectly manageable without it for the `providers` - I've done it couple of times and it is very little work.

**Review by ashb — APPROVED** [review]

**Pull request merged**
